packages/tools/src/tools/displayTools/Labelmap/labelmapDisplay.ts

```diff
@@ -117,7 +109,8 @@ async function addSegmentationRepresentation(
  */
 function removeSegmentationRepresentation(
   toolGroupId: string,
-  segmentationRepresentationUID: string
+  segmentationRepresentationUID: string,
+  immediate = false
```

Comment:
If this is about rendering let's make that clear in the parameter name, like renderImmediately